src/payment-account.js

```diff
@@ -13,6 +15,60 @@ class PaymentAccount extends ExchangePaymentAccount {
       return trade;
     });
   }
+
+  sell (bankId) {
+    return super.sell(bankId).then(trade => {
+      console.log('*** SELL TRADE CREATED ***', trade);
+      return trade;
+    });
+  }
+
+  add (obj) {
+    assert(obj, 'bank obj is required');
+
+    const b = {
+      account: {
+        currency: obj.account.currency,
+        bic: obj.account.bic,
+        number: obj.account.number
+      },
+      holder: {
+        name: obj.holder.name,
+        address: {
+          street: obj.holder.address.street,
+          city: obj.holder.address.city,
+          zipcode: obj.holder.address.zipcode,
+          country: obj.holder.address.country
+        }
+      },
+      bank: {
+        address: {
+          country: obj.bank.address.country,
+          street: obj.bank.address.street || null,
+          zipcode: obj.bank.address._ipcode || null,
+          city: obj.bank.address.city || null
+        }
+      }
+    };
+    return this._api.authPOST('bank-accounts', b).then(res => {
+      return new PaymentAccount(this._api, undefined, undefined, res);
+    });
+  }
+
+  getAll (quote) {
```

Comment:
This should use a class method on `Bank`. E.g.

```js
Bank.getAll(this._api).then((accounts =>
```

Maybe part of the confusion is in having `PaymentAccount` as well as `Bank`. So maybe instead of what I said just before, you should move everything from the old `bank.js` into `PaymentAccount` instead. Or make Bank a subclass of PaymentAccount.